### PR TITLE
LPS-190004 Replace aui:button with clay:button in asset-publisher-web

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-190004)

In this issue we want to replace all usages of aui:button with clay:button in asset-publisher-web module, ideally it is going to allow us improve the accessibility.

## Test Scenario 1 (add_asset_selector.jsp)

* Create a new basic web content.
* Create a widget page
* Add an asset publisher to this page and configure it
* Select manual collection
* Add another scope
* Save and close the modal
* Click on the addd button of the asset publisher and select “Add content (Select Scope and type)”
* Check the buttons

Pasted Graphic 10

## Test Scenario 2 (asset_list.jsp)

* Create a new basic web content.
* Create a widget page
* Add an asset publisher to this page and configure it
* Select manual collection
* Go to select colección and check the buttons

Pasted Graphic 11

## Test Scenario 2 (source.jsp)

* Create a new basic web content.
* Create a widget page
* Add an asset publisher to this page and configure it
* Select dynamic collection
* Go to source and check the button

Pasted Graphic 12

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/add_asset_selector.jsp [hunk 1]
```diff
@@ -108,9 +108,18 @@ String redirect = PortalUtil.getLayoutFullURL(layout, themeDisplay);
...
+		<clay:button
+			displayType="primary"
+			label="add"
+			onClick='<%= liferayPortletResponse.getNamespace() + "addAssetEntry();" %>'
+		/>
+
+		<clay:link
+			displayType="secondary"
+			href="<%= redirect %>"
+			label="cancel"
```
Comment (claraizquierdo) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/add_asset_selector.jsp:120]: Hi @BarbaraCabrera , I believe we should translate the labels.

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Hi @BarbaraCabrera , everything seems to be working fine. The only thing I found is that we should translate the labels in all the `clay:button`

UPDATE: This is not necessary because the taglib translate it. Sorry for the confusion.

**Review by claraizquierdo — APPROVED** [review]

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/138454

**Pull request closed without merging**